pandas/_libs/groupby.pyx

```diff
@@ -54,10 +56,10 @@ cdef inline float64_t median_linear(float64_t* a, int n) nogil:
         n -= na_count
 
     if n % 2:
-        result = kth_smallest_c( a, n / 2, n)
+        result = kth_smallest_c( a, n // 2, n)
```

Comment:
Space after open paren

Reply:
I'm not sure if it changes with language-level, but in cython `/` defaults to cdivision, which behaves like python floor-division (... mostly.  its corner cases are bizarre).  The cython docs says there is a non-trivial perf impact of disabling cdivision.  It might be worth checking how it treats floordiv.

Then again, this isn't an op we're doing zillions of times, so not a huge deal.

Reply:
If you try to just use / here compilation will fail as you try to assign a double to an int